locales/deDE.lua

```diff
@@ -1004,7 +1004,7 @@ for key,value in pairs({
 	[6751] = "Sonderbare fruchtbeladene Pflanze",	-- Strange Fruited Plant
 	[6752] = "Sonderbare wedelbestückte Pflanze",	-- Strange Fronded Plant
 	[7510] = "Sprießender Wedel",	-- Sprouted Frond
-	--TODO: [12564] = "Assassination Notice",	-- Assassination Notice
+	[12564] = "Auftragsmordnotiz",	-- Assassination Notice
```

Comment:
Where did you find the translation for objects in this table?

Reply:
I used the german wowhead for most of them, example https://www.wowhead.com/de/item=12564/auftragsmordnotiz
As wowhead sniffs the data with their client from german players that use it on the german localization blizzard offers, it's translated 1:1 as it should be

Reply:
I see, it's just that those are objects (interactable things you click in game), not an item. Sometimes they have corresponding names though. But in this case this looks like a mistake on our side, if there's an item with the exact same id, I doubt there's an object too...
Thanks, I'll take a closer look at these.

Reply:
@Nelythia, I could only find "Mogu Chest" object in addition to "Assassination Notice" item on Wowhead. So if you can share how I can find the rest I'd appreciate it (to add to other languages).

Reply:
Sure. I basically used a method that I came up with as I was trying to find the objects on wowhead as well.
I'll explain it thoroughly as an example with the Gorishi Hive Hatchery and then add the source web pages for the rest of them.

- [174793] = "Brutstätte der Gorishi",	-- Gorishi Hive Hatchery
I referenced the requirement of the associated quest item's spell, https://www.wowhead.com/spell=13630/scraping which says "Requires: Gorishi Hive Hatchery" in the 'Quick Facts' on the top right, refering to the objects name as far as I understand. However wowhead does not have the object cached in an actual page itself, so I can't confirm if the ID is correct, for example. 
It's more of circumstancial evidence that made me translate it that way. But I did in fact just notice that I apparently didn't add the full objects name in deDE, which should be "Schwarmbrutstätte der Gorishi". I'll fix it some time later with more translations, in case you accept the method I used as proper way to translate those.
DE: https://www.wowhead.com/de/spell=13630/kratzen

- [177684] = "Stein von Shy-Rotam",	-- Stone of Shy-Rotam
Same way of referencing the actual quest item's spell requirement of "Stone of Shy-Rotam" in the 'Quick Facts'
EN: https://www.wowhead.com/classic/spell=16796/summon-shy-rotam
DE: https://www.wowhead.com/classic/de/spell=16796/shy-rotam-beschw%C3%B6ren

- [180327] = "Kohlenpfanne des Wahnsinns",	-- Brazier of Madness
Didn't have a 'Quick Facts' requirement listed, but the translation was both the same on associated item & spell
EN: https://www.wowhead.com/classic/spell=24264/extinguish
DE: https://www.wowhead.com/classic/de/spell=24264/l%C3%B6schen

- [186278] = "Brennendes Wrack",	-- Burning Wreckage
Manual translation, the only information I could find on the web was that it was spawned in Dustwallow Marsh some time pre-cata, but couldn't find any page or quest information.

- [186322] = "Monument der Familie Hyal",	-- Hyal Family Monument
Using the 'Quick Facts' requirement
EN: https://www.wowhead.com/spell=42390/lay-wreath
DE: https://www.wowhead.com/de/spell=42390/kranz-niederlegen

- [191136] = "Durchtränkter Mutterboden",	-- Soaked Fertile Dirt
A german fanpage database references this translation. I also found a YouTube video of the quest on enUS localization, which happens to have a mouseover of the object, named "Soaked Fertile Dirt", at the same spot that the german fanpage has as one of the possible spawn points.
EN Video with Mouseover: https://youtu.be/4K71AOgK33g?t=83
DE fanpage database: https://wow.freierbund.de/map_and_guide/314331/Durchtraenkter_Mutterboden.html

I admit it's a bit messy to use as translation, but without proper wowhead pages that have the object cached, it was the best way I could think of to give it translations that seem to match properly.

Reply:
Thank you! Yes, it is very time-consuming to research everything manually and requires some language knowledge because for example in Russian I won't be able to just take these lines due to different grammatical cases... For Russian I know what to do but not for other languages, unfortunately.